src/lib.rs

```diff
@@ -1064,20 +1068,21 @@ impl Mock {
     /// let _m = mock("GET", "/").with_body("hello world").create();
     /// ```
     ///
-    pub fn create(self) -> Self {
+    #[must_use]
```

Comment:
I don't think this does what you intend. This ensures someone who _calls_ `create()` uses the result (`Self`), but doesn't require that `create()` itself is called. And since the intended pattern is to bind the `Mock` to an unused variable (`let _m = mock(...);`) that satisfies the `must_use` requirement, so normal usage won't see this.

For example:

```
mock("GET", "/").with_status(200); // No warning
mock("GET", "/").with_status(200).create(); // Warning: must_use
let _m = mock("GET", "/").with_status(200); // No warning
let _m = mock("GET", "/").with_status(200).create(); // No warning
```

Arguably all the builder methods on `Mock` should be `#[must_use]`, but it still won't ensure that `.create()` is called.